File: practice_mvvm/Paging/app/src/main/java/com/miraclehwan/paging/model/Item.java
package com.miraclehwan.paging.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Item {
    @SerializedName("owner")
    @Expose
    public Owner owner;
    @SerializedName("is_accepted")
    @Expose
    public boolean isAccepted;
    @SerializedName("score")
    @Expose
    public long score;
    @SerializedName("last_activity_date")
    @Expose
    public long lastActivityDate;
    @SerializedName("creation_date")
    @Expose
    public long creationDate;
    @SerializedName("answer_id")
    @Expose
    public long answerId;
    @SerializedName("question_id")
    @Expose
    public long questionId;
    @SerializedName("last_edit_date")
    @Expose
    public long lastEditDate;
}

File: Paging3/app/src/main/java/com/miraclehwan/paging/response/Item.kt
package com.miraclehwan.paging.response


import com.google.gson.annotations.SerializedName

data class Item(
    @SerializedName("answer_id")
    val answerId: Int,
    @SerializedName("content_license")
    val contentLicense: String,
    @SerializedName("creation_date")
    val creationDate: Int,
    @SerializedName("is_accepted")
    val isAccepted: Boolean,
    @SerializedName("last_activity_date")
    val lastActivityDate: Int,
    @SerializedName("last_edit_date")
    val lastEditDate: Int,
    @SerializedName("owner")
    val owner: Owner,
    @SerializedName("question_id")
    val questionId: Int,
    @SerializedName("score")
    val score: Int
)